deployability/README.MD

```diff
@@ -1,4 +1,4 @@
-## Deployability general documentation
+## JobFlow - Deployability general documentation
```

Comment:
JobFlow is a module within deployability, we should remove it from here

Reply:
The developed workflow tool (initially called `Workflow engine` (orchestrator) is a composition of multiple modules), has been renamed to `JobFlow`

These modules can work without the orchestrator but they are part of the JobFlow package

Reply:
Modified in commit https://github.com/wazuh/wazuh-qa/pull/5460/commits/8b07c4235ad62ada42e8e8e0eae015a7a465538b